DatadogSessionReplay/Tests/Recorder/Utilties/ImageDataProviderTests.swift

```diff
@@ -24,6 +24,9 @@ class ImageDataProviderTests: XCTestCase {
     }
 
     func test_returnsValidStringAndIdentifier_WhenContentIsValid() throws {
+        if #available(iOS 17, *) {
+            throw XCTSkip("TODO: RUM-2969 Fix image hashing tests for iOS 17+")
+        }
```

Comment:
The `image.pngData()` gives different results between iOS 17 and prior versions (results seem to be stable between iOS 16 and 15). I was unable to solve this problem with refactoring the code due to too much dependency on static variables and `UIImage` extension. Bigger refactoring looked risky as this code involves threading assumptions that I have no full expertise on. Behaviour Team's eye on this looks required.

Ultimately, I decided to skip these tests on iOS 17 until the problem is addressed by Behaviour Team in `RUM-2969`.